services/centralized-kubecost/0.37.1/release/release.yaml

```diff
@@ -11,7 +11,7 @@ spec:
         kind: HelmRepository
         name: mesosphere.github.io-charts-stable
         namespace: kommander-flux
-      version: 0.37.1
+      version: 0.37.2
```

Comment:
this helm chart version doesn't exist! and I think it's fine not to bump the app version, we just need it to be bumped between dkp versions if there are any changes, and we've already bumped this app from 2.6, so we should be good to keep it at the current app version even if we make config changes